src/data/ACTIONS/root/SMN.ts

```diff
@@ -1,58 +1,87 @@
 import {Attribute} from 'event'
-import {PETS} from '../../PETS'
 import {ensureActions} from '../type'
 
 // use action id of a skill id in the group to avoid potential duplications
 export const SMN_COOLDOWN_GROUP = {
```

Comment:
This was only ever imported by ACN, is it still needed?

Reply:
To clarify, are you asking if the Pets data file should be removed?  I hadn't thought about that, but I can try removing it too and see if nothing breaks.  There are SCH and MCH pets in that file as well, so I'll run some of those parses as well to ensure there's not some hidden reference to them.

Reply:
`SMN_COOLDOWN_GROUP` specifically, which isn't part of the diff, but since it was only imported by the ACN data I don't think it's needed anymore? GH gets weird with how it shows comments outside the actual diff lines.